test/functionalTest/cases/2763_postUpdateContext_error_return/invalid_service_path_gives_400_bad_request.test

```diff
@@ -0,0 +1,56 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I can't find one. I have created the directory but there is no functest inside ...
Also can't find any modified functest for this ...

Must check this, understand what's happened here.